.changeset/short-hornets-clap.md

```diff
@@ -0,0 +1,6 @@
+---
+'@rainbow-me/rainbowkit': minor
+'example': patch
```

Comment:
`example` patch not needed. Please reference recent connector addition changesets formatting and example

Reply:
Sorry about that, I had updated the `example` to test out the vercel builds and added it while running `changeset` by mistake, reverted.